pkg/util/pravegacluster.go

```diff
@@ -81,8 +82,42 @@ func ConfigMapNameForSegmentstore(clusterName string) string {
 	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
 }
 
-func StatefulSetNameForSegmentstore(clusterName string) string {
-	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
+//function to check if the version is below 0.7 or not
+func IsVersionBelow07(ver string) bool {
+	if ver == "" {
+		return true
+	}
+	first3 := strings.Trim(ver, "\t \n")[0:3]
```

Comment:
Why can't we use CompareVersions here? Lets try to reuse existing methods wherever possible, Its is not good to use only way of doing things here and another way in the webhook.

Reply:
changed it